src/Variable.cpp

```diff
@@ -200,6 +200,26 @@ VariablePtr Variable::subvariable(Space space, std::string_view baseName, Space
   }
 }
 
+VariablePtr Variable::subvariable(Space space, Space shift) const
```

Comment:
Can this not be expressed with the name-based method rather than duplicating the body of the function?

Reply:
I introduced a subfunction to handle the two cases in a533edd46731861420fe2620693aba67b65d642d